### PR TITLE
Cart manage

### PR DESCRIPTION
**Pull request by KamiyaGaikwad**
Added cart functionality
1. add to the cart
2. remove from the cart
3. increase and decrease the quantity

### CODE REVIEW

File: src/Reducer/cartReducer2.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+
+export function cartReducer(state, action) {
+  
+    switch (action.type) {
+      case "ADD_TO_CART"://increment quantity
```
Comment (mansipadgelwar) [inline src/Reducer/cartReducer2.js:5]: remove console logs

File: src/Reducer/cartReducer2.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  //       if (products.find(product => product._id === action.payload)) {
+  //         let newState1 = [...state];
+
+  //         // increment cart item quantity if exists
+  //         if (newState1.find(item => item.product._id === action.payload._id)) {
+  //           
+  //           ++newState1.find(item => item.product.sku === action.payload.sku)
+  //             .quantity;
+  //         } else {
+  //           newState1.push({
```
Comment (mansipadgelwar) [inline src/Reducer/cartReducer2.js]: Remove it if not needed

**Review by mansipadgelwar — APPROVED** [review]

File: src/Reducer/cartReducer2.js (hunk 1 above)
Comment (KamiyaGaikwad) [reply src/Reducer/cartReducer2.js:5]: Removed it. Thank you

File: src/Reducer/cartReducer2.js (hunk 2 above)
Comment (KamiyaGaikwad) [reply src/Reducer/cartReducer2.js]: Removed unnecessary comments. Thank you

**Review by vinc3nati — APPROVED** [review]

**Pull request merged**